travel-sponsorship.md

```diff
@@ -0,0 +1,82 @@
+# Kubernetes Contributor Travel Fund
+
+## Purpose
+
+To establish and administer a fund for members of the Kubernetes upstream contributor community to travel and particiate in KubeCon/CloudNativeCon.
+
+## Requirements
+
+ - Candidates must be a [member of the Kubernetes GitHub organization][membership] or working towards it. Exceptions: TODO
+ - Those requesting funds must indicate that they do not have funding available from another source, such as an employer or the event itself that might cover costs for presenters.
+
+## Process
+
+Submit an issue to a Kubernetes Travel Fund Issue-Only repository. 
+Your issue will go into a private repository where only TODO: $certain_group can see your information due to privacy.
+
+Requests must be made three months before the event. Requests submitted after that will not be approved.
```

Comment:
Can we say that the travel fund coordinators will come up with a deadline that is approximately three months before the event. And that the requests should be submitted before the dead line. (give some wiggle room!)